packages/schematics/angular/module/index.ts

```diff
@@ -64,23 +64,25 @@ function addDeclarationToNgModule(options: any): Rule {
 }
 
 export default function (options: any): Rule {
-  options.module = normalizePath(options.module);
+  return (host: Tree, context: SchematicContext) => {
+    options.module = findModuleFromOptions(host, options);
```

Comment:
This should only be called if `options.module` has a value... modules aren't automatically found.